### ISSUE TITLE
ENCODE project search HTTPS problems

### ISSUE THREAD

**Issue by olegs** [JetBrains-Research/jbr#85]
On JDK11 I get some errors with https://encodeproject.org.
```
[Dec 19, 2019 15:35:54] Markup folder: /Users/oleg/.jbr_browser/genomes
[Dec 19, 2019 15:35:54] Caches directory: /Users/oleg/.jbr_browser/cache
[Dec 19, 2019 15:35:54] Updating annotations '/Users/oleg/.jbr_browser/genomes'...
[Dec 19, 2019 15:35:54] Loading plugins...
[Dec 19, 2019 15:35:55] Loaded plugin: Overlap BED tracks
[Dec 19, 2019 15:35:55] Loaded plugin: BED files tuning
[Dec 19, 2019 15:35:55] Loaded plugin: SPAN support
[Dec 19, 2019 15:35:55] Loaded plugin: Search ENCODE support
[Dec 19, 2019 15:35:55] Loaded plugin: JBR base tracks
[Dec 19, 2019 15:35:55] Successfully loaded plugins: 5
[Dec 19, 2019 15:35:55] Launching browser...
[Dec 19, 2019 15:35:55] Loading supported genomes config: /Users/oleg/.jbr_browser/genomes/annotations.yaml ...
[Dec 19, 2019 15:35:55] Supported genomes: ce10, ce11, ce6, danRer10, danRer11, danRer7, dm3, dm6, hg18, hg19, hg38, hs37-1kg, hs37d5, mm10, mm9, rn5, rn6
[Dec 19, 2019 15:35:55] Plugin Overlap BED tracks loaded 1 extension point(s) UIExtensionPoint
[Dec 19, 2019 15:35:55] Plugin BED files tuning loaded 1 extension point(s) UIExtensionPoint
[Dec 19, 2019 15:35:55] Plugin SPAN support loaded 1 extension point(s) UIExtensionPoint
[Dec 19, 2019 15:35:55] Plugin Search ENCODE support loaded 1 extension point(s) UIExtensionPoint
[Dec 19, 2019 15:35:55] Loading genes /Users/oleg/.jbr_browser/genomes/hg19/Homo_sapiens.GRCh37.87.gtf.gz...
[Dec 19, 2019 15:35:56] Loading genes /Users/oleg/.jbr_browser/genomes/hg19/Homo_sapiens.GRCh37.87.gtf.gz: done in 1.182 s
[Dec 19, 2019 15:35:57] Plugin BED files tuning loaded 1 extension point(s) TrackViewExtensionPoint
[Dec 19, 2019 15:35:57] Plugin SPAN support loaded 1 extension point(s) TrackViewExtensionPoint
[Dec 19, 2019 15:35:57] Plugin JBR base tracks loaded 3 extension point(s) TrackViewExtensionPoint
[Dec 19, 2019 15:35:58] Initializing browser: Preprocessing 1 tracks
[Dec 19, 2019 15:35:58] Preprocessing track 'Ensembl Genes'...
[Dec 19, 2019 15:35:58] Preprocessing track 'Ensembl Genes': done in 10.01 ms
[Dec 19, 2019 15:35:58] Preprocessing: 100% (1/1), Elapsed time: 14 ms [done]
[Dec 19, 2019 15:36:27] ERROR SearchEncodeUtil Exception
Caused by: ERROR SSLPeerUnverifiedException peer not authenticated
javax.net.ssl.SSLPeerUnverifiedException: peer not authenticated
	at java.base/sun.security.ssl.SSLSessionImpl.getPeerCertificates(SSLSessionImpl.java:526)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy.verifySession(SSLIOSessionStrategy.java:205)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy$1.verify(SSLIOSessionStrategy.java:188)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.doHandshake(SSLIOSession.java:354)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.isAppInputReady(SSLIOSession.java:503)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:120)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:588)
	at java.base/java.lang.Thread.run(Thread.java:834)

[Dec 19, 2019 15:36:27] ERROR SearchEncodeUtil Exception
Caused by: ERROR SSLPeerUnverifiedException peer not authenticated
javax.net.ssl.SSLPeerUnverifiedException: peer not authenticated
	at java.base/sun.security.ssl.SSLSessionImpl.getPeerCertificates(SSLSessionImpl.java:526)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy.verifySession(SSLIOSessionStrategy.java:205)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy$1.verify(SSLIOSessionStrategy.java:188)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.doHandshake(SSLIOSession.java:354)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.isAppInputReady(SSLIOSession.java:503)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:120)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:588)
	at java.base/java.lang.Thread.run(Thread.java:834)

[Dec 19, 2019 15:36:27] ERROR SearchEncodeUtil Exception
Caused by: ERROR SSLPeerUnverifiedException peer not authenticated
javax.net.ssl.SSLPeerUnverifiedException: peer not authenticated
	at java.base/sun.security.ssl.SSLSessionImpl.getPeerCertificates(SSLSessionImpl.java:526)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy.verifySession(SSLIOSessionStrategy.java:205)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy$1.verify(SSLIOSessionStrategy.java:188)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.doHandshake(SSLIOSession.java:354)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.isAppInputReady(SSLIOSession.java:503)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:120)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:588)
	at java.base/java.lang.Thread.run(Thread.java:834)

[Dec 19, 2019 15:36:27] ERROR SearchEncodeUtil Exception
Caused by: ERROR SSLPeerUnverifiedException peer not authenticated
javax.net.ssl.SSLPeerUnverifiedException: peer not authenticated
	at java.base/sun.security.ssl.SSLSessionImpl.getPeerCertificates(SSLSessionImpl.java:526)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy.verifySession(SSLIOSessionStrategy.java:205)
	at org.apache.http.nio.conn.ssl.SSLIOSessionStrategy$1.verify(SSLIOSessionStrategy.java:188)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.doHandshake(SSLIOSession.java:354)
	at org.apache.http.nio.reactor.ssl.SSLIOSession.isAppInputReady(SSLIOSession.java:503)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:120)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:588)
	at java.base/java.lang.Thread.run(Thread.java:834)

[Dec 19, 2019 15:36:59] open -R /Users/oleg/.jbr_browser/logs/jbr.log

```

**Comment by dievsky** [comment]
Hmm, really strange. I assume that you're able to open https://encodeproject.org/ in a web browser with no warnings?

**Comment by dievsky** [comment]
Closing this since we've closed the duplicate #92 .
